releases.md

```diff
@@ -21,12 +21,24 @@ breadcrumbs:
 For more information on the changes planned for each release, please see the [Project Roadmap](https://github.com/orgs/opensearch-project/projects/1)
 
 Note:  We have not added a major release to the 2023 schedule yet.  If/when we add one, it will replace a minor release in the 2.x line.  See below for criteria for a major releases.
-
-
-| Release Number| First RC Generated (release window opens) | Latest Possible Release Date (release window closes) | Release Manager | Tracking Issue |
-|:--------------|:-----------------|:---------------|:---------------|:---------------|
-| 2.12.0        | February 6th, 2024    | February 20th, 2024  | [Prudhvi Godithi](https://github.com/prudhvigodithi) | [4115](https://github.com/opensearch-project/opensearch-build/issues/4115) |
-| Being finalized   | TBD    | TBD | TBD | [Proposal - 186](https://github.com/opensearch-project/.github/issues/186) |
+  
+
+| Release Number| First RC Generated (release window opens) | Latest Possible Release Date (release window closes) | Release Manager                                     | Tracking Issue |
+|:--------------|:------------------------------------------|:---------------|:----------------------------------------------------|:---------------|
+| 2.12.0        | February 6th, 2024                        | February 20th, 2024  | [Prudhvi Godithi](https://github.com/prudhvigodithi) | [4115](https://github.com/opensearch-project/opensearch-build/issues/4115) |
+|1.3.15 | Feb 27 2024                               | March 05 2024|                                                     |[4294](https://github.com/opensearch-project/opensearch-build/issues/4294)
+2.13.0 | March 19th, 2024                          | April 2nd, 2024
```

Comment:
In the markdown preview this table isn't rendering correctly, I think its due to missing `|` characters.

Reply:
Whats your environment setup like? When I pull this it's rendering alright

Reply:
Maybe the website markdown is processed differently that GitHub's preview, see how it looks here:
https://github.com/opensearch-project/project-website/pull/2588/files?short_path=1d2b91d#diff-1d2b91d957f8612017f196e6cc2e83c33bcfe3fd348dc3619e9e9672e797d46f

Reply:
Ah that is it indeed it seems. Going to push through then because the Jekyll rendered version seems okay.